helpers/settings.go

```diff
@@ -115,3 +136,43 @@ func (s *Settings) InitSettings(envVars EnvVars) error {
 
 	return nil
 }
+
+func getRedisSettings(env *cfenv.App) (string, string, error) {
+	uri, err := getRedisService(env)
+	if err != nil {
+		uri = os.Getenv("REDIS_URI")
+	}
+	if uri == "" {
+		uri = "redis://localhost:6379"
+	}
+
+	u, err := url.Parse(uri)
```

Comment:
That Parse method returns a tuple (two things) and the first one is the parsed uri which is set to `u` and the second one is an error set to `err`